Update Logs (#1506)
zeit/docs: pages/docs/v2/platform/deployments.mdx

You can find a full list of the commands available in the [Now CLI Reference](/docs/now-cli/), along with the deployments section of the [Now API Reference](/docs/api/#endpoints/deployments).

## Logs

There are three types of logs available, **Build Time**, **Runtime**, and **Edge Network**.

### Build Time

**Build Time** logs are generated during the [build step](/docs/v2/build-step/). These logs contain information about the build process and are stored **indefinitely**.

### Edge Network

> **Edge Network** logs are generated when requesting a path from the Edge. These logs contain information about a request to a specific path with details such as the path name, request method, and status code. These logs are **not persisted**.

Reviewer: This is slightly confusing, since the Serverless Functions might also run on the edge. It's fine for now, however.

### Runtime

> **Runtime** logs are generated by [Serverless Functions](/docs/v2/serverless-functions/introduction) whilst the application is running. Runtime logs are stored in memory only as they arrive from the Serverless Function and are **not persisted**.

Reviewer: ```suggestion
**Runtime** logs are generated by [Serverless Functions](/docs/v2/serverless-functions/introduction) while they're being invoked. Runtime logs are stored in memory only as they arrive from the Serverless Function and are **not persisted**.
```

> The only exception to this is **failed requests**. If a request leads to the Serverless Function throwing an error, the log for this will be stored indefinitely whereas all other Runtime logs will be lost when navigating away from the page.

Reviewer: ```suggestion
The only exception to this are **failed requests**. If a request leads to the Serverless Function throwing an error, the log for this will be stored indefinitely whereas all other Runtime logs will be lost when navigating away from the page.
```
